Networking/WebSockets/BuiltInWebSocket.cc

```diff
@@ -448,16 +442,6 @@ namespace litecore { namespace websocket {
                 return;
             }
             
-            // CBL-1633: Need to check for EOF here, or _socket->write returns
-            // 0 endlessly
-            if(n == 0 && _socket->atWriteEOF()) {
```

Comment:
Note: If you look carefully at the code above this, you'll see that this block is unreachable (n cannot be zero); so I removed it.